dts/binding-template.yaml

```diff
@@ -7,6 +7,13 @@ description: |
 # Used to map nodes to bindings
 compatible: "manufacturer,device"
 
+# Set this to a truthy value if this binding is describing a node that
+# has a struct device representation within the Zephyr device
```

Comment:
Could make it "... has a 'struct device' representation ..."

Was wondering what kind of device a "struct device" is.